props/saber_fett263_buttons.h

```diff
@@ -2627,8 +2548,15 @@ SaberFett263Buttons() : PropBase() {}
       style_parser.GetArgument(current_preset_.GetStyle(blade_num_), set_num_ + 2, argspace);
       calc_ = strtol(argspace, NULL, 0);
       IntEdit::SetIntValue(calc_);
-      sound_library_.SaySelect();
       if (NUM_BLADES == 1) blade_num_ = 1;
+      switch (set_num_) {
```

Comment:
Add a comment explaining why we only use SaySelect() some of the time.

Reply:
added